### PR TITLE
Disable caching project _meta

### PR DESCRIPTION
**Pull request by Vogtinator**
latest_updated is broken for _meta:
https://github.com/openSUSE/open-build-service/issues/6323

### CODE REVIEW

**Comment by coolo** [comment]
We use TTL_DUPLICATE for that - aka very short caching.

**Comment by Vogtinator** [comment]
Sure.

**Comment by coolo** [comment]
Not sure what you mean with 'Sure.', but having disabled code is no solution.

**Comment by Vogtinator** [comment]
By "Sure." I mean that I changed it and pushed the change.

File: osclib/cache.py [hunk 1]
```diff
@@ -99,16 +99,17 @@ class Cache(object):
...
-        # Handles clearing local cache on package deletes. Lots of queries like
-        # updating project info, comment, and package additions.
-        r'/source/([^/]+)/(?:[^/?]+)(?:\?[^/]+)?$': TTL_LONG,
         # Presumably users are not interweaving in short windows.
         r'/statistics/latest_updated': TTL_SHORT,
+        # Use TTL_DUPLICATE for project _meta as only description changes are listed in latest_updated:
+        # https://github.com/openSUSE/open-build-service/issues/6323
+        r'/source/([^/]+)/_meta$': TTL_DUPLICATE,
+        ## Handles clearing local cache on package deletes. Lots of queries like
+        ## updating project info, comment, and package additions.
```
Comment (jberry-suse) [inline osclib/cache.py]: Assume the double comment was unintentional?

Comment (Vogtinator) [reply osclib/cache.py]: It was intentional - I commented out the entire block. The comment refers to the commented out code, so IMO this makes it obvious that it's not relevant to the active code anymore.

**Pull request merged**